docs/plugins.md

```diff
@@ -4,7 +4,7 @@ Helm 2.1.0 introduced the concept of a client-side Helm _plugin_. A plugin is a
 tool that can be accessed through the `helm` CLI, but which is not part of the
 built-in Helm codebase.
 
-Existing plugins can be found on [related](related.md#helm-plugins) section or by searching [Github](https://github.com/search?q=topic%3Ahelm-plugin&type=Repositories).
+Existing plugins can be found on [related](https://docs.helm.sh/related/#helm-plugins) section or by searching [Github](https://github.com/search?q=topic%3Ahelm-plugin&type=Repositories).
```

Comment:
Even though docs.helm.sh rewrites the URL incorrectly, lets use the proper fragment format without the last slash `https://docs.helm.sh/related#helm-plugins`

Reply:
Also can you squash this into one commit? Thanks!